INFO  | 03/31/2020 03:54:33 PM | File run_exp.py           | Line 26    | Writing log file to exp/GRAN\GRANMixtureBernoulli_transport_2020-Mar-31-15-54-32_16080\log_exp_16080.txt
INFO  | 03/31/2020 03:54:33 PM | File run_exp.py           | Line 27    | Exp instance id = 16080
INFO  | 03/31/2020 03:54:33 PM | File run_exp.py           | Line 28    | Exp comment = None
INFO  | 03/31/2020 03:54:33 PM | File run_exp.py           | Line 29    | Config =
INFO  | 03/31/2020 03:54:33 PM | File gran_runner.py       | Line 124   | Train/val/test = 8/2/2
INFO  | 03/31/2020 03:54:33 PM | File gran_runner.py       | Line 137   | No Edges vs. Edges in training set = 2.0
ERROR | 03/31/2020 03:54:36 PM | File run_exp.py           | Line 42    | Traceback (most recent call last):
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\run_exp.py", line 40, in main
    runner.test()
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\runner\gran_runner.py", line 310, in test
    A_tmp = model(input_dict)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\parallel\data_parallel.py", line 150, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 477, in forward
    A = self._sampling(batch_size)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 361, in _sampling
    node_state_in.view(-1, H), edges, edge_feat=att_edge_feat, node_pos=node_pos)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 122, in forward
    state = self._prop(state, edge, edge_feat=edge_feat, node_pos=node_pos, layer_idx=ii)
  File "c:\Users\olive\OneDrive\Dokumenter\GitHub\GRAN\model\gran_mixture_bernoulli.py", line 89, in _prop
    msg = self.msg_func[layer_idx](edge_input)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\container.py", line 92, in forward
    input = module(input)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\modules\linear.py", line 87, in forward
    return F.linear(input, self.weight, self.bias)
  File "C:\Users\olive\Anaconda3\envs\Bachelor\lib\site-packages\torch\nn\functional.py", line 1369, in linear
    ret = torch.addmm(bias, input, weight.t())
RuntimeError: size mismatch, m1: [0 x 256], m2: [258 x 128] at C:/w/1/s/tmp_conda_3.7_055457/conda/conda-bld/pytorch_1565416617654/work/aten/src\THC/generic/THCTensorMathBlas.cu:273

